analyzer/pkg/apk/apk.go

```diff
@@ -55,14 +55,9 @@ func (a alpinePkgAnalyzer) parseApkInfo(scanner *bufio.Scanner) (pkgs []types.Pa
 			}
 			pkg.Version = version
 		case "o:":
-			origin := string(line[2:])
-			originPkg := types.Package{
-				Name:    origin,
-				Version: version,
-			}
-			if analyzer.CheckPackage(&originPkg) {
-				pkgs = append(pkgs, originPkg)
-			}
+			origin := line[2:]
```

Comment:
Is the `string` cast not necessary?

Reply:
Yes, `line` was originally `string`. IDE told me that the case is unnecessary 😄